### DISCUSSION TITLE
Regarding Deployment

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8218] [Q&A] [answered]
Can the deployment be centralized or automated from workspace 1 ?

Also, Please help me to know whether any Ports are associated or not ?

Is there any Telemetry Data Collected, Will any Sensitive and Personal Information will collected by the application ?

Thanks

**Comment by phoerious** [comment, marked as answer]
Please do not create duplicate issues. https://github.com/keepassxreboot/keepassxc/issues/8211#issuecomment-1171229853
